htdocs/class/libraries/vendor/xoops/xmf/src/Module/Helper/AbstractHelper.php

```diff
@@ -137,8 +137,8 @@ public function addLog($log)
      */
     protected function serializeForHelperLog($value)
     {
-        if (is_resource($value) || !is_null(@get_resource_type($value))) {
-            $value = '(resource)';
+        if (is_resource($value)) {
+            $value = '(' . get_resource_type($value) . ')';
```

Comment:
Would it be a bit clearer what you are looking at like this?
`$value = '(resource:' . get_resource_type($value) . ')';`

Reply:
Actually, I need to push this through XMF, so I'll have thay ready soon.